test/performance/scripts/run-tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+c9_gql_url="http://localhost:3001/graphql"
+c9_wallet_address="http://localhost:3000/accounts/gfranklin"
+hlb_wallet_address="http://localhost:4000/accounts/pfry"
+
+# Verify that the localenv backend is live
+if curl -s --head --request GET "$c9_gql_url" | grep "HTTP/1.[01]" > /dev/null; then
+  echo "Localenv is up: $c9_gql_url"
+else
+  echo "Localenv is down: $c9_gql_url"
+  exit 1
+fi
+
+# Verify that cloud nine wallet address is live
+if curl -s --head --request GET "$c9_wallet_address" | grep "HTTP/1.[01]" > /dev/null; then
```

Comment:
So this is kinda funny....

The outgoing payment metrics should show after all. [I was wrong here](https://github.com/interledger/rafiki/pull/2828#discussion_r1704511996) and the peer-to-peer bruno tests DO populate the outgoing payment based metrics (I was just impatient - underestimated the delay). So the performance tests should populate them too.

You can confirm this beyond the metrics if you do `docker logs rafiki-cloud-nine-mock-ase-1` after running the tests and you should see many of these:

```
Error: No account found for wallet address
    at handleOutgoingPaymentCreated (/home/rafiki/localenv/mock-account-servicing-entity/build/index.js:2200:11)
    ...
```

This is showing the mock ase cant find the wallet address in the handle [outgoing payment create webhook handler](https://github.com/interledger/rafiki/blob/main/localenv/mock-account-servicing-entity/app/lib/webhooks.server.ts#L64-L130) and then it doesnt deposit liquidity and the payment doesnt get funded. I looked into the DB and saw that we had 2 `Grace Franklin` wallet addresses. `https://cloud-nine-wallet-backend/accounts/gfranklin` and `https://localhost:3000/accounts/gfranklin`. Looks like the `localhost:3000` one is getting created by the [mock ase's wallet address not found handler](https://github.com/interledger/rafiki/blob/main/localenv/mock-account-servicing-entity/app/lib/webhooks.server.ts#L256-L282) when we ping here. Then when the mock ASE looks up the wallet address, it expects this new one while the one specified in the performance test script is the original so it never gets funded.

Perhaps we can ping something else? Or nothing? `http://cloud-nine-wallet-backend:3000/accounts/gfranklin` connects but creates a new record as well, and we cant reach by `https://cloud-nine-wallet-backend/accounts/gfranklin`. Originally I noticed I could change the script to do `const CLOUD_NINE_WALLET_ADDRESS = 'https://localhost:3000/accounts/gfranklin'` but this is just kinda hacking around the unintended wallet address creation.

Reply:
metrics showing in the dashboard setting `constCLOUD_NINE_WALLET_ADDRESS = 'https://localhost:3000/accounts/gfranklin'` in the script

![image](https://github.com/user-attachments/assets/f45ef237-b9b8-4024-9aad-bc894dfb36a6)